test/unit/org/apache/cassandra/cql3/CQLTester.java

```diff
@@ -521,6 +535,27 @@ protected static void addVirtualKeyspace()
         VirtualKeyspaceRegistry.instance.register(SystemViewsKeyspace.instance);
     }
 
+    protected void clearSchema()
```

Comment:
most of my tests start with this pattern

```
qt().check(rs -> {
  clearSchema();
  ...
});
```

The `check` block gets called 1k times by default.... so needed a simple way to reset all state